app_builder/lib/app_builder/macos.ex

```diff
@@ -147,7 +147,16 @@ defmodule AppBuilder.MacOS do
 
     let releaseScriptPath = Bundle.main.path(forResource: "rel/bin/mac_app", ofType: "")!
 
+    let resourcePath = Bundle.main.resourcePath ?? ""
+
+    var environment = ProcessInfo.processInfo.environment
+    let path = environment["PATH"] ?? ""
+    let elixirBinPath = "\\(resourcePath)/rel/elixir/bin"
+
+    environment["PATH"] = "\\(elixirBinPath):\\(path)"
```

Comment:
So `app_builder` is meant to be generic, so we need to pass any additional path as an option. Maybe we should have a `additional_paths: ["rel/elixir/bin"]` configuration that we will traverse and add here?